lib/ZcapClient.js

```diff
@@ -140,6 +140,9 @@ export class ZcapClient {
     capability, controller, invocationTarget, expires,
     allowedActions = []
   } = {}) {
+    if(!controller) {
+      throw new TypeError('"controller" must be provided.');
```

Comment:
We may as well get as specific as some of the other checks here:

```suggestion
    if(!(typeof controller === 'string' && controller.includes(':'))) {
      throw new Error(
        '"controller" must be a string expressing an absolute URI.');
    }      
```

Reply:
Approval conditional on the above being applied.

Reply:
Updated [83439f1](https://github.com/digitalbazaar/ezcap/pull/16/commits/83439f13b15d0f84ac5f491b19c7b0b2117064f6) and added changelog [2c5bd2c](https://github.com/digitalbazaar/ezcap/pull/16/commits/2c5bd2c7ccb183481356cc286b497f9bcae1b9d5)